docs/spectator/lang/cpp/meters/monotonic-counter.md

```diff
@@ -0,0 +1,25 @@
+A Monotonic Counter (float) is used to measure the rate at which an event is occurring, when the
```

Comment:
Given it is essentially the same concept, I'm not sure a separate page is needed for this. I would suggest just one for monotonic counters that mentions both variants.

Reply:
The other languages have them split into separate pages, because the uint type was added after the fact, so it was adding a new page across the board rather than refactoring.

I like this suggestion - the other languages would need to have this refactor done as well.

Reply:
Agree with leaving it as-is for now, to get this feature out the door, and then following up sometime in the future to refactor.

I think consistency across the client documentation is useful.